frontend/src/bundles/search-candidates/pages/candidates.tsx

```diff
@@ -162,16 +162,30 @@ const Candidates: React.FC = () => {
                     <Grid
                         className={getValidClassNames(
                             styles.searchResults,
+                            filteredCandidates.length > 0 && styles.scroll,
                             isFilterOpened ? styles.searchResultsHidden : '',
                         )}
                     >
-                        {filteredCandidates.map((candidate) => (
-                            <CandidateProfile
-                                key={candidate.id}
-                                isProfileCard
-                                candidateData={candidate}
-                            />
-                        ))}
+                        {filteredCandidates.length > 0 ? (
```

Comment:
```suggestion
                        {!filteredCandidates.length ? (
```

Reply:
these changes will cause eslint unicorn/no-negated-condition and unicorn/explicit-length-check errors.